### PR TITLE
Quick Access Google

### PR DESCRIPTION
**Pull request by iamrahulmahato**
### 🛠️ Fixes Issue

 #168

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by ridsuteri — COMMENTED** [review]
@iamrahulmahato the extension is going blank, please look into this :(

**Comment by iamrahulmahato** [comment]
@ridsuteri recently i  checked its working
Screenshot 2022-04-13 191133

**Comment by ridsuteri** [comment]
Screenshot (7)
Well, this is how it looks on my system :(

**Comment by iamrahulmahato** [comment]
@ridsuteri its working on chrome but not in edge i think some google compatibility issue is there

File: Quick Access Google/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+window.onload = () => {
+    chrome.management.getAll((apps) => {
```
Comment (ridsuteri) [inline Quick Access Google/script.js:2]: maybe because of these?

**Comment by ridsuteri** [comment]
> @ridsuteri its working on my chrome but not in edge i think some google compatibility issue is there

Try looking out for fixes and do connect with me via discord DM incase you are unable to fix this one

**Comment by iamrahulmahato** [comment]
@ridsuteri tried a lot and finally, I got ans that Edge and other browser is based on Chromium, so it can run most chrome extensions. When we run this extension in Edge or another, it shows a blank, because Edge doesn't have a chrome app

It works only in the chrome browser

**Review by ridsuteri — APPROVED** [review]
@iamrahulmahato approving this with minor changes in the readme :)
Good work rahul !

**Comment by iamrahulmahato** [comment]
@ridsuteri thank you

**Pull request merged**